modules/brainyBidAdapter.js

```diff
@@ -0,0 +1,131 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'brainy';
+const BASE_URL = '//proparm.co.jp/ssp/p/pbjs';
+
+/**
+ * Check if the browser supports flash
+ * 0 is return if it dosen't support flash
+ * @return      {int} Flash version
+ */
+/**
+ * 接続元のブラウザがフラッシュに対応しているか判定
+ * 対応していなければ0を返す
+ * @return      {int} フラッシュのバージョン
+ */
+function _getFlash() {
+  try {
+    var _mac = (navigator.userAgent.indexOf('Mac') != -1);
+    if (document.all) {
+      if (_mac) {
+        if (window['sample']) {
+          return ((window['sample'].FlashVersion() & 0xffff0000) >> 16);
+        }
+      } else {
+        var _axo = new ActiveXObject('ShockwaveFlash.ShockwaveFlash');
+        return Math.floor(_axo.FlashVersion() / 0x10000);
+      }
+    } else {
+      if (navigator.plugins && navigator.plugins['Shockwave Flash']) {
+        var info = navigator.plugins['Shockwave Flash'].description.split(' ');
+        var _v = parseInt(info[2]);
+        if (!isNaN(_v)) {
+          return _v;
+        }
+      }
+    }
+  } catch (e) {}
+  return 0;
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Check if the bid account ID and slotID is valid
+   * @param {object} bid the brainy bid to validate
+   * @return {boolean}
+   */
+  /**
+   * adUnits.bidに値が入っているかを判断する
+   * @param  {object} bid 検証する入札リクエスト
+   * @return {boolean}
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.acountID && bid.params.slotID);
```

Comment:
This looks like a typo.  should all `acountId` be `accountId`?  

Also, can you submit corresponding docs pull-request to https://github.com/prebid/prebid.github.io ?